continuous_integration/environment-3.10.yaml

```diff
@@ -68,5 +68,8 @@ dependencies:
   - mmh3
   - jinja2
   - pip
+  # The nightly pyarrow / arrow-cpp packages currently don't install with latest
```

Comment:
Is this still a problem, I assume you checked but asking just in case.

Reply:
Thanks for reviewing @ncclementi. Yeah, this is still an issue. I initially forgot to transfer this change over to the 3.10 environment file and saw the solve issue from https://github.com/dask/dask/pull/9465 show up